packages/components/src/components/contract-card/contract-card-items/vanilla-options-card-body.tsx

```diff
@@ -27,25 +45,26 @@ const VanillaOptionsCardBody = ({ contract_info, currency, getCardLabels, is_sol
                     </ContractCardItem>
 
                     <ContractCardItem header={ENTRY_SPOT}>
-                        <Money amount={entry_spot_display_value} />
+                        {entry_spot_display_value && <Money amount={entry_spot_display_value} />}
                     </ContractCardItem>
 
                     <ContractCardItem header={STRIKE}>
-                        <Money amount={barrier} />
+                        {barrier && <Money amount={barrier} />}
                     </ContractCardItem>
                 </div>
                 <ContractCardItem
                     className='dc-contract-card-item__total-profit-loss'
                     header={TOTAL_PROFIT_LOSS}
                     is_crypto={isCryptocurrency(currency)}
-                    is_loss={+profit < 0}
-                    is_won={+profit > 0}
+                    is_loss={profit !== undefined && +profit < 0}
```

Comment:
This is in case if `profit === 0`? Then if profit is number why do we need `+profit`?

Reply:
We don't need, it's an old code, I'll remove it